zulipterminal/model.py

```diff
@@ -558,6 +558,17 @@ def update_subscription(self, event: Event) -> None:
 
                 self.controller.update_screen()
 
+    def toggle_stream_muted_status(self, button: Any) -> Tuple[bool, str]:
```

Comment:
Done :+1:

Reply:
@neiljp In the generic PopUpView it would benefit to pass a button instead of any specific data (like `stream_id`), hence it was the choice to pass the button itself here.